code/game/objects/items/holotape.dm

```diff
@@ -158,6 +158,11 @@
 		var/turf/T = get_turf(src)
 		C.loc = T
 
+/obj/item/holotape/Bumped(var/mob/living/silicon/S)
```

Comment:
in a perfect world this would only allow Silicons through. It won't.
You're going to get every single possible atomic structure in the universe that could possibly bump the holotape in this proc, which is of course dangerous, you need to ensure the bumpee (S in this case) is indeed a silicon.
The proc for that is ```istype(S, /mob/living/silicon)``` or ```issilicon(S)```